build.gradle

```diff
@@ -460,6 +461,34 @@ artifacts {
     }
 }
 
+// publishing
+publishing {
+    publications {
+        maven(MavenPublication) {
+            artifact source: jar
+            artifact source: sourcesJar, classifier: "src"
+            artifact source: devJar, classifier: "dev"
+            if (apiPackage) {
+                archives source: apiJar, classifier: "api"
+            }
+        }
+    }
+    
+    repositories {
+        maven {
+            String owner = System.getenv("REPOSITORY_OWNER") ?: "Unknown"
```

Comment:
I guess therese properties are provided by the github CI? What about installation to local maven? Should there be any fallback?

Reply:
it is a configuration for publishing to github, so for this block it shouldn't matter and definitely should not influence how local packages are published. If any it's a problem of the other block where I configure group id, etc